### DISCUSSION TITLE
Purchased the wrong license after being directed to the site

### DISCUSSION THREAD

**Discussion by CookingDaddy** [aclap-dev/video-downloadhelper#992] [License & Purchase requests]
### What's your issue?

It was only after I had been charged for the firefox version of this service I realized that the link ZI had followed was the wrong one and I need the  Chrome version. I need a  refund so I can purchase the correct version

### The 10 first digits (only 10) of your license number or email address

3d2e94bf3a

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
